src/test/groovy/pl/tomaszdziurko/jvm_bloggers/InitialBlogDataPopulationTriggerSpec.groovy

```diff
@@ -0,0 +1,42 @@
+package pl.tomaszdziurko.jvm_bloggers
+
+import pl.tomaszdziurko.jvm_bloggers.blog_posts.BlogPostsFetchingScheduler;
+import pl.tomaszdziurko.jvm_bloggers.blog_posts.domain.BlogPostRepository
+import pl.tomaszdziurko.jvm_bloggers.blogs.BloggersDataFetchingScheduler;
+import pl.tomaszdziurko.jvm_bloggers.blogs.domain.BlogRepository;
+import spock.lang.Specification;
+import spock.lang.Subject
+
+class InitialBlogDataPopulationTriggerSpec extends Specification {
+
+    def BlogRepository blogRepository = Mock()
```

Comment:
Either `def` or `BlogRepository`

I like this approach:

    BlogRepository blogRepository = Mock(BlogRepository)